canonical-web-and-design/maas-ui
# Create Pod actions and reducers using createSlice

## Done

- Replaced Pod actions and reducers with createSlice.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)

### QA steps

- Make sure you can still list, create, delete, update, compose etc. (via /r/kvm).
